.azure-pipelines.yml

```diff
@@ -148,6 +148,27 @@ stages:
         AZURE_ACCESS_TOKEN: "$(System.AccessToken)"
         TFLAGS: ""
 
+  - job: http_only_brotli_zstd
+    displayName: ubuntu HTTP only with zstd and brotli
+    timeoutInMinutes: 20
+    pool:
+      vmImage: 'ubuntu-latest'
+    steps:
+    - script: sudo apt install stunnel4 python-impacket libzstd-dev libbrotli-dev
+      displayName: 'apt install'
+
+    - script: ./buildconf && ./configure --disable-dict --disable-file --disable-ftp --disable-gopher --disable-imap --disable-ldap --disable-pop3 --disable-rtmp --disable-rtsp --disable-scp --disable-sftp --disable-smb --disable-smtp --disable-telnet --disable-tftp
+      displayName: 'configure disable non-http'
+
+    - script: make
+      displayName: 'make'
+
+    - script: make test-nonflaky
+      displayName: 'test'
+      env:
+        AZURE_ACCESS_TOKEN: "$(System.AccessToken)"
+        TFLAGS: ""
+
```

Comment:
Any reason for adding a new job as opposed to combining it with another test?

Reply:
Just I was afraid broken anything...

Reply:
I'd probably reuse an existing job to reduce the carbon footprint, but it's up to @bagder :)

Reply:
I agree with @Lekensteyn, we should rather combine this into an existing job. Not only the carbon footprint wins, we also get the complete results done faster.

Reply:
I agree with you, but I prefer you choose the existing job to add zstd.

I must admit my focus was on zstd code, makefile and test and not the better CI solution... :-)